habitat/datasets/eqa/mp3d_eqa_dataset.py

```diff
@@ -52,6 +58,19 @@ def __init__(self, config: Config = None) -> None:
         with gzip.open(config.DATA_PATH.format(split=config.SPLIT), "rt") as f:
             self.from_json(f.read())
 
+        questions_vocabulary = {
```

Comment:
This PR is big enough, but the question stuff here for EQA needs to be fixed.  The questions are a list of tokens, not an individual token themself.

Reply:
Yes, EQA is moving to separate PR. It was here to test the change end-to-end.